variables.tf

```diff
@@ -1,17 +1,11 @@
-variable "tenant_name" {
+variable "tenant" {
   type = string
   description = "The tenant of your control plane. Ex.: mytenant"
 }
 
 variable "control_plane" {
   type = string
-  description = "Control plane address. Ex.: mytenant.cyral.com"
-}
-
-variable "control_plane_port" {
```

Comment:
As I look at this, I'm thinking it's a breaking change. Although _we_ don't presently use this, some user of this provider may. 

I see this repo has only been alive for about 4 days, and it hasn't been released yet, so I'm thinking this should be fine although it's breaking. After we do the first release, though, we'll need to start following [this](https://www.terraform.io/docs/extend/best-practices/versioning.html).

Reply:
I fully agree, the thing is, for now we don't even have a version hahaha, so semantic versioning becomes a little tough.